packages/shared/src/components/profile/ProfileForm.tsx

```diff
@@ -97,37 +87,38 @@ export default function ProfileForm({
 
   const onSubmit = async (event: FormEvent): Promise<void> => {
     event.preventDefault();
-    setDisableSubmit?.(true);
```

Comment:
I think we need to disable the button while submitting so the user will not double click on it

Reply:
Ah valid point indeed!